### ISSUE TITLE
can not reproduce the eval accuracy by demo code

### ISSUE THREAD

**Issue by ltm920716** [huggingface/open-r1#530]
hi，
  I just run the demo code in readme bellow, using 4*4090 gpus：
https://github.com/huggingface/open-r1?tab=readme-ov-file#training-models
```
accelerate launch --config_file=recipes/accelerate_configs/zero3.yaml src/open_r1/sft.py \
    --model_name_or_path Qwen/Qwen2.5-1.5B-Instruct \
    --dataset_name open-r1/OpenR1-Math-220k \
    --packing \
    --max_seq_length 16384 \
    --per_device_train_batch_size 4 \
    --gradient_checkpointing \
    --bf16 \
    --output_dir data/Qwen2.5-1.5B-Open-R1-Distill
```

and then I eval the AIME 2024, only get 0.08,  the  model generates tokens repeatedly
```
s blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that a specific blue vertex is blue is 1/2, and red is 1/2. Therefore, the probability that 
```

so is there something wrong? or could you update a kindly demo for reproducing？ thanks

**Comment by juliancodaforno** [comment]
Same here I get 0.067 on AIME 2024 by using the config_demo.yaml with 8 A100 GPUs: 
accelerate launch --config_file recipes/accelerate_configs/zero3.yaml src/open_r1/sft.py \
    --config recipes/Qwen2.5-1.5B-Instruct/sft/config_demo.yaml

Or should we increase the number of epochs to reproduce the results?

**Comment by zhangmxxx** [comment]
Same issue, and the evaluation with `max_new_tokens:4096` on AIME 2024 is taking a lot of time, due to the repeated output.

**Comment by linkedlist771** [comment]
Hi, did you solve this? I met the same problem, the GRPO's training downgrade the model performance.

**Comment by ltm920716** [comment]
not yet，maybe should add more training data

**Comment by linkedlist771** [comment]
This may relate with the reward weights, by default, they equally contribute to the reward, while the accuracy reward is harder to learn.  I have increased its weight, which improves the accuracy a little but still downgrading.
